docs/administration.mesh-policies.md

```diff
@@ -20,14 +20,18 @@ section of the navigation area. After clicking policies, you can view all polici
 Policy creation is only possible for a user who has a partner admin role. There is a 'Create Policy' button top
 right on the meshPolicies screen. After clicking on it, there is a redirection to the screen of "Create Policy" where all details need to be filled out.
 
-- **Policy Name**: choose a human readabale display name for the meshPolicy
+- **Policy Name**: choose a human readable display name for the meshPolicy
+
+- **Description (optional)**: add a description to make everyone aware what your policy is about and why it is necessary.
```

Comment:
f: Please remove `.` at the end (description and howToFix) or add the `.` to all other items.